Lib/asyncio/events.py

```diff
@@ -155,6 +155,10 @@ def close(self):
         """Coroutine to wait until service is closed."""
         return NotImplemented
 
+    def get_loop(self):
+        """ Get the event loop the Server object is attached to."""
+        return NotImplemented
```

Comment:
Please add a test for make sure that `AbstractServer().get_loop()` returns `NotImplemented`.

Reply:
@asvetlov It should raise `NotImplementedError`, please fix.  `NotImplemened` singleton is for comparisons in Python.

Reply:
Oooh, my bed.